mfjcs-rest/config.yml

```diff
@@ -2,3 +2,5 @@ logging:
   level: INFO
   loggers:
     org.mfjcs: DEBUG
+
+storageBasePath: ~/.mfjcs-dev/store
```

Comment:
Ah. I should run the jar... have only run unit tests so far. Same with the
solr setup

On Wed, 24 Aug 2016 10:20 Marc Portier, <notifications@github.com> wrote:

> In mfjcs-rest/config.yml
> <https://github.com/marc-portier/mfjcs/pull/3#discussion_r76012690>:
>
> > @@ -2,3 +2,5 @@ logging:
> >    level: INFO
> >    loggers:
> >      org.mfjcs: DEBUG
> > +
> > +storageBasePath: ~/.mfjcs-dev/store
>
> this yields an error?
>
> $ java -jar target/mfjcs-rest-1.0-SNAPSHOT.jar server config.yml
> config.yml has an error:
>   * Unrecognized field at: storageBasePath
>     Did you mean?:
>       - solrZkHost
>       - server
>       - logging
>       - metrics
>       - solrItemCollection
>         [1 more]
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/marc-portier/mfjcs/pull/3/files/a9dbe4de4a65060068a13809d530a398c07cf6e0#r76012690>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAJVvfveQU8Jup66UEbEs8T2H5CtTi_Jks5qi_68gaJpZM4JrXRl>
> .
>